.github/workflows/aur-git.yml

```
@@ -1,49 +1,49 @@
+      - "README*.md"
+      - "docs/**"
+      - ".github/**"
+      - "!.github/workflows/aur-git.yml"
+  workflow_dispatch:
+
+jobs:
+  aur-git-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)